integration-tests/cypress/e2e/project/edit/formdescriptioneditor.cy.js

```diff
@@ -176,4 +176,53 @@ describe('/projects/:projectId/edit - FormDescriptionEditor', () => {
+    cy.getByTestId('treeitem-contextmenu').findByTestId('new-object').click();
+    cy.getByTestId('childCreationDescription').children('[role="button"]').invoke('text').should('have.length.gt', 1);
+    cy.getByTestId('childCreationDescription').click().get('[data-value="Widgets Link Description"]').should('exist').click();
+    cy.getByTestId('create-object').click();
+    cy.getByTestId('Is Enabled Expression').should('not.exist');
+    cy.getByTestId('GroupDescription-more').click();
+    cy.getByTestId('treeitem-contextmenu').findByTestId('new-object').click();
+    cy.getByTestId('childCreationDescription').children('[role="button"]').invoke('text').should('have.length.gt', 1);
+    cy.getByTestId('childCreationDescription').click().get('[data-value="Widgets Label Description"]').should('exist').click();
+    cy.getByTestId('create-object').click();
+    cy.getByTestId('Is Enabled Expression').should('not.exist');
+    // Editable widgets should have isEnabledExpression
+    cy.getByTestId('GroupDescription-more').click();
+    cy.getByTestId('treeitem-contextmenu').findByTestId('new-object').click();
+    cy.getByTestId('childCreationDescription').children('[role="button"]').invoke('text').should('have.length.gt', 1);
+    cy.getByTestId('childCreationDescription').click().get('[data-value="Widgets Button Description"]').should('exist').click();
+    cy.getByTestId('create-object').click();
+    cy.getByTestId('Is Enabled Expression').should('exist');
+    cy.getByTestId('GroupDescription-more').click();
+    cy.getByTestId('treeitem-contextmenu').findByTestId('new-object').click();
+    cy.getByTestId('childCreationDescription').children('[role="button"]').invoke('text').should('have.length.gt', 1);
+    cy.getByTestId('childCreationDescription').click().get('[data-value="Widgets Flexbox Container Description"]').should('exist').click();
+    cy.getByTestId('create-object').click();
+    cy.getByTestId('Is Enabled Expression').should('exist');
+    cy.getByTestId('GroupDescription-more').click();
+    cy.getByTestId('treeitem-contextmenu').findByTestId('new-object').click();
+    cy.getByTestId('childCreationDescription').children('[role="button"]').invoke('text').should('have.length.gt', 1);
+    cy.getByTestId('childCreationDescription').click().get('[data-value="Toolbar Actions Button Description"]').should('exist').click();
+    cy.getByTestId('create-object').click();
+    cy.getByTestId('Is Enabled Expression').should('exist');
+    cy.getByTestId('GroupDescription-more').click();
+    cy.getByTestId('treeitem-contextmenu').findByTestId('new-object').click();
+    cy.getByTestId('childCreationDescription').children('[role="button"]').invoke('text').should('have.length.gt', 1);
+    cy.getByTestId('childCreationDescription').click().get('[data-value="Widgets Slider Description"]').should('exist').click();
+    cy.getByTestId('create-object').click();
+    cy.getByTestId('Is Enabled Expression').should('exist');
```

Comment:
It's not specific to this code or this PR, but I'll never understand why when we write test code it becomes OK, or even natural, so create code duplication. :smile:

Reply:
You're right, it should be mutualized.